int32 simState
spring_boxes/UnityObjData[2] objDataList
spring_boxes/SystemParams systemParams
float64 timeDelta